.github/CODEOWNERS

```diff
@@ -16,19 +16,19 @@
 *   @open-telemetry/specs-semconv-maintainers @open-telemetry/specs-semconv-approvers
 
 # Schemas and schema file tooling
-/schemas/ @tigrannajaryan
-/internal/tools/schema_check.sh @tigrannajaryan
+/schemas/ @tigrannajaryan @open-telemetry/specs-semconv-maintainers @open-telemetry/specs-semconv-approvers
```

Comment:
This is a good example IMHO https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS

Reply:
https://github.com/orgs/open-telemetry/teams/specs-semconv-approvers includes the maintainers as per https://github.com/open-telemetry/community/blob/main/docs/how-to-configure-new-repository.md#collaborators-and-teams so it can be omitted here I think, yes.